modules/gdnative/videodecoder/resource_importer_av_gdnative.cpp

```diff
@@ -0,0 +1,96 @@
+/*************************************************************************/
+/*  resource_importer_gdnative.cpp                                       */
```

Comment:
The comment should be updated to reflect this file's name (note also the trailing `*/` to close the box).